tests/test_rmmember.py

```diff
@@ -0,0 +1,39 @@
+import logging
+
+import teachers_gitlab.main as tg
+
+def test_remove_member(mock_gitlab):
+    entries = [
+        {'login': 'alpha'},
+    ]
+
+    mock_gitlab.register_project(42, 'student/alpha', members =
+    [
```

Comment:
Would you, please, reformat this by indenting (and possibly keeping the `[` on previous line)?

```
def test_remove_member(mock_gitlab):
    ...
    mock_gitlab.register_project(42, 'student/alpha', members=[
        {
            ...
        },
        {
            ...
        },
    ])
```

Thanks!